apps/media-viewer/src/Mediaviewer.vue

```diff
@@ -207,20 +207,19 @@ export default {
         a: 1
       })
 
+      this.activeMediaFile.path = this.activeMediaFile.path.replace(/^\//, '');
       if (this.publicPage()) {
         const path = [
           '..',
-          'dav',
-          'public-files',
```

Comment:
This will break on public shares

Reply:
a closer look implies that for some reason `this.publicPage()` is true for non public files all the time in reva/ocis

Reply:
Is something missing in ocs @butonic ?